docs/widget.rst

```diff
@@ -253,16 +253,29 @@ See also the documentation of the ``resource_registry`` argument in
 Specifying Widget Requirements
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When creating a new widget, you may specify its requirements by using
-the ``requirements`` attribute:
+When instantiating a new widget, you may specify its requirements by using the ``requirements`` attribute.
+The requirements are specified as a sequence of two-tuples, dicts, or a combination of both two-tuples and dicts.
+
+The two-tuple form is used by all core Deform widgets and uses the resource registry.
```

Comment:
If you look carefully at this PR, you can spot that this is not true anymore. I realize now should have mentioned that explicitly in the PR, as you may very well disagree.

Reply:
Good catch! Fixed in 8d25cc9.

Reply:
Oh, and by the way, I agree. But then I realized that you changed the format for `fileupload`, but did not remove it from the default resource registry. Was that an oversight or deliberate?

Reply:
sounds like an oversight